### PR TITLE
Touchups

### PR DESCRIPTION
**Pull request by aredridel**

### CODE REVIEW

File: lib/serializer/properties/writer.js [hunk 1]
```diff
@@ -59,6 +59,9 @@ ReadStream.prototype._process = function (namespace, data) {
                 data.forEach(function (item) {
                     this._process(namespace, item);
                 }.bind(this));
+            } else if (data === null) {
```
Comment (tlivings) [inline lib/serializer/properties/writer.js]: Does `undefined` case matter here?

**Comment by totherik** [comment]
Looks like there's some whitespace issues in `lib/transcoder.js`, specifically, [this](https://github.com/aredridel/spud/blob/c1540b17875ec75cf7f610ed27564cf2726ddd2c/lib/transcoder.js#L75-L112) method.

Edit: actually, in a lot of places.

**Comment by aredridel** [comment]
Whitespace errors fixed.

**Comment by totherik** [comment]
I think this LGTM

**Comment by jasisk** [comment]
Spoke to @aredridel in person about some stuff. Looks good.

**Pull request merged**